package.json

```diff
@@ -48,11 +49,12 @@
     "cover": "istanbul cover node_modules/.bin/_mocha -- -u exports --compilers js:babel/register -R spec",
     "coveralls": "NODE_ENV=test ./node_modules/.bin/istanbul cover ./node_modules/mocha/bin/_mocha --report lcovonly -- -u exports --compilers js:babel/register -R spec && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
```

Comment:
I took that from the coveralls site, if there's a better way, let's have it.

On Thu, Aug 13, 2015 at 8:57 AM, Daniel Cousens <notifications@github.com>
wrote:

> In package.json
> <https://github.com/JedWatson/react-select/pull/373#discussion_r36945543>:
>
> > @@ -48,11 +49,12 @@
> >      "cover": "istanbul cover node_modules/.bin/_mocha -- -u exports --compilers js:babel/register -R spec",
> >      "coveralls": "NODE_ENV=test ./node_modules/.bin/istanbul cover ./node_modules/mocha/bin/_mocha --report lcovonly -- -u exports --compilers js:babel/register -R spec && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
>
> @bruderstein <https://github.com/bruderstein> why not just | coveralls by
> the way?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JedWatson/react-select/pull/373/files#r36945543>.
>